### ISSUE TITLE
iOS: App crashed when pick image in 4G network

### ISSUE THREAD

**Issue by juncaixinchi** [Sh1d0w/multi_image_picker#284]
**Bug Report**

`multi_image_picker` version: 4.5.6
Is it happening on iOS: Yes

```
$ flutter doctor -v
[✓] Flutter (Channel stable, v1.9.1+hotfix.4, on Mac OS X 10.14.6 18G87, locale zh-Hans-CN)
    • Flutter version 1.9.1+hotfix.4 at /Users/wisnuc-imac/flutter
    • Framework revision cc949a8e8b (3 weeks ago), 2019-09-27 15:04:59 -0700
    • Engine revision b863200c37
    • Dart version 2.5.0


[!] Android toolchain - develop for Android devices (Android SDK version 27.0.3)
    • Android SDK at /Users/wisnuc-imac/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    ✗ Flutter requires Android SDK 28 and the Android BuildTools 28.0.3
      To update using sdkmanager, run:
        "/Users/wisnuc-imac/Library/Android/sdk/tools/bin/sdkmanager" "platforms;android-28" "build-tools;28.0.3"
      or visit https://flutter.dev/setup/#android-setup for detailed instructions.
    ✗ Android licenses not accepted.  To resolve this, run: flutter doctor --android-licenses

[✓] Xcode - develop for iOS and macOS (Xcode 11.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.1, Build version 11A1027
    • CocoaPods version 1.7.4

[!] Android Studio (version 3.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-915-b08)

[✓] VS Code (version 1.29.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 2.25.1

[✓] Connected device (1 available)
    • iPhone • 23deadc0330c3a5179e15e281e7ab06be308b265 • ios • iOS 13.1.2
```

**Describe the bug**
It works well when network is on WiFi. chang network to 4G, try pick images, then app crashed.

xcode logs: 
![a](https://user-images.githubusercontent.com/8253814/66823211-5dc7cd00-ef78-11e9-8110-8b157a732bb8.png)

**Comment by juncaixinchi** [comment]
@Sh1d0w is there any progress ?

**Comment by Sh1d0w** [comment]
Hi, this was fixed in version 4.5.9

**Comment by juncaixinchi** [comment]
> Hi, this was fixed in version 4.5.9

thanks
